continuous_integration/environment-3.10.yaml

```diff
@@ -0,0 +1,56 @@
+name: dask-distributed
+channels:
+  - conda-forge
+  - defaults
```

Comment:
It doesn't seem possible to test pytorch on Python 3.10 until https://github.com/pytorch/pytorch/issues/66424 is addressed. Perhaps a reminder issue for testing pytorch once it has 3.10 support would be suitable?

Reply:
Good catch. Typically we have three different test builds.

- One cythonized, mostly vanilla dependencies
- One vanilla dependency build
- One with all optional build dependencies

In the past these were distributed on py3.7, 3.8 and 3.9, s.t. python 3.9 installed all dependencies.

While this test matrix still has flaws, it's a pragmatic approach and I'd like to keep it this way.
If the optional dependencies didn't make it to py3.10, yet, we should test them in py3.9 and use the py3.10 as the vanilla build, i.e. the environment.yaml should be similar to the 3.8 one